examples/UdpNtpClient/UdpNtpClient.ino

```diff
@@ -0,0 +1,174 @@
+  Serial.begin(115200);
+  while (!Serial);
+
+  Serial.println("\nStarting UdpNtpClient with NativeEthernet using QNEthernet Library");
+
+  // start the Ethernet connection:
+  Serial.println("Initialize Ethernet with DHCP:");
+
+#if USING_DHCP
+  // Using DHCP
+  Serial.print("Using DHCP => ");
+  Ethernet.begin();
+#else  Ethernet.setDNSServerIP(mydnsServer);
+  
+  // Using static IP
+  Serial.print("Using static IP => ");
+  Ethernet.begin(myIP, myNetmask, myGW);
+#endif
+
+  if (!Ethernet.waitForLocalIP(5000))
+  {
+    Serial.println("Failed to configure Ethernet");
+
+    if (!Ethernet.linkStatus())
+    {
+      Serial.println("Ethernet cable is not connected.");
+    }
+
+    // Stay here forever
+    while (true)
+    {
+      delay(1);
+    }
+  }
+  else
+  {
+    Serial.print("IP Address = ");
+    Serial.println(Ethernet.localIP());
+  }
+  
+  Udp.begin(localPort);
+}
+
+void loop() 
+{
+  sendNTPpacket(timeServer); // send an NTP packet to a time server
+
+  // wait to see if a reply is available
+  delay(1000);
+  
+  if (Udp.parsePacket()) 
+  {
+    // We've received a packet, read the data from it
+    Udp.read(packetBuffer, NTP_PACKET_SIZE); // read the packet into the buffer
+
+    // the timestamp starts at byte 40 of the received packet and is four bytes,
+    // or two words, long. First, extract the two words:
+
+    unsigned long highWord = word(packetBuffer[40], packetBuffer[41]);
+    unsigned long lowWord = word(packetBuffer[42], packetBuffer[43]);
+    // combine the four bytes (two words) into a long integer
+    // this is NTP time (seconds since Jan 1 1900):
+    unsigned long secsSince1900 = highWord << 16 | lowWord;
+    Serial.print("Seconds since Jan 1 1900 = ");
+    Serial.println(secsSince1900);
+
+    // now convert NTP time into everyday time:
+    Serial.print("Unix time = ");
+    // Unix time starts on Jan 1 1970. In seconds, that's 2208988800:
+    const unsigned long seventyYears = 2208988800UL;
+    // subtract seventy years:
+    unsigned long epoch = secsSince1900 - seventyYears;
+    // print Unix time:
+    Serial.println(epoch);
+
+    // print the hour, minute and second:
+    Serial.print("The UTC time is ");       // UTC is the time at Greenwich Meridian (GMT)
+    Serial.print((epoch  % 86400L) / 3600); // print the hour (86400 equals secs per day)
+    Serial.print(':');
+    
+    if (((epoch % 3600) / 60) < 10) 
+    {
+      // In the first 10 minutes of each hour, we'll want a leading '0'
+      Serial.print('0');
+    }
+    
+    Serial.print((epoch  % 3600) / 60); // print the minute (3600 equals secs per minute)
+    Serial.print(':');
+    
+    if ((epoch % 60) < 10) 
+    {
+      // In the first 10 seconds of each minute, we'll want a leading '0'
+      Serial.print('0');
+    }
+    
+    Serial.println(epoch % 60); // print the second
+  }
+  
+  // wait ten seconds before asking for the time again
+  delay(10000);
+  Ethernet.maintain();
```

Comment:
`Ethernet.maintain()` is no longer needed. I know this is a balancing act, balancing the Arduino API with new features, but I think removing this call is okay.

Reply:
It's better to remove this.